caikit/runtime/grpc_server.py

```diff
@@ -84,6 +84,7 @@ def __init__(
             thread_pool=self.thread_pool,
             interceptors=(PROMETHEUS_METRICS_INTERCEPTOR,),
             options=(self.config.runtime.grpc.options or {}).items(),
+            maximum_concurrent_rpcs=self.config.runtime.grpc.maximum_concurrent_rpcs,
```

Comment:
Interesting that this does not go into the `options` arg? Assuming this is correct, then this looks good to me

Reply:
Yeah its kinda annoying. TBH I thought this was an option for the longest time